### DISCUSSION TITLE
Suppress dependency lookup warning in PRs

### DISCUSSION THREAD

**Discussion by wyardley** [renovatebot/renovate#17818] [Request Help] [answered]
Is there either a way to ignore a specific dependency that's generating a "dependency lookup warning" without entirely filtering it (i.e., allow renovate to still detect it, but don't complain about the lookup warning if it can't get it for that specific dependency), and / or suppress the new warning in downstream PRs about existing dependency lookup warnings?

(context: https://github.com/renovatebot/renovate/issues/15651)

**Comment by rarkins** [comment, marked as answer]
If you add it to `ignoreDeps` or set up a `packageRules` entry to disable it, then Renovate won't attempt to look it up and then it won't trigger a dependency lookup warning.

**Reply by wyardley** [reply]
Thanks. And other than by manager, is there a way to further dial in circle orb namespaced? I see it show up as `foo` vs `slug/foo` so not sure how to make sure I don’t ignore `someorg/foo` along with `myorg/foo` (that said, I think I found a workaround that does what I need, sort of)

**Reply by rarkins** [reply]
matchPackageNames and matchPackagePatterns provide a lot of granularity for this

**Reply by wyardley** [reply]
I guess what I mean is that renovate doesn’t seem to “see” the slug at all for orbs AFAICT

**Reply by rarkins** [reply]
Sorry, would need a reproduction repo in order to understand what you mean. e.g. do orbs have a separate depName and packageName perhaps?

**Reply by wyardley** [reply]
Good question, I'll take a look and / or setup a repro repo. Separate question from the original question, so I'll start a new topic if needed. 👍

**Reply by wyardley** [reply]
Also, you're right - the namespace is in the `packageName`, but the renovate dashboard just showed the `depName`. So should be possible to filter on `packageName` if I need to

```json
            "depName": "foo",
            "datasource": "orb",
            "packageName": "someorg/foo",
```
